src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -107,7 +107,7 @@ const Body = BaseForm.extend({
           Logger.error(`Authenticator is not listening on port ${currentPort}.`);
           if (countFailedPorts === ports.length) {
             Logger.error('No available ports. Loopback server failed and polling is cancelled.');
-            this.options.appState.trigger('invokeAction', 'authenticatorChallenge.cancel-polling');
+            this.options.appState.trigger('invokeAction', 'authenticatorChallenge-cancel');
```

Comment:
Before this change, this action path is composed with `authenticatorChallenge` and the name value within the `cancel`. Does that mean we won't look at the name anymore with the change?

Reply:
We should just look at the names of the actions that idx-js provides within the object and trigger events with those names accordingly.

Reply:
Make sense. But I wonder if the name within is redundant.